common/modifiers/dvg_lot_modifiers.txt

```diff
@@ -0,0 +1,37 @@
+﻿modifier_army_dissaproves = {
+    icon = gfx/interface/icons/timed_modifier_icons/modifier_rifle_positive.dds
+    interest_group_approval_add = -15
+    country_authority_add = -100
+}
+
+modifier_army_purged = {
+    icon = gfx/interface/icons/timed_modifier_icons/modifier_rifle_positive.dds
+    ig_armed_forces_pol_pop_str_mult = -0.5
+   ig_armed_forces_pol_pop_str_mult = -0.5
+   country_authority_add = -100
+}
+
+modifier_army_bolstered = {
+    icon = gfx/interface/icons/timed_modifier_icons/modifier_rifle_positive.dds
+   ig_armed_forces_pol_pop_str_mult = 0.5
```

Comment:
Duplicate, and should be either:
interest_group_ig_armed_forces_pop_attraction_mult to attract more pops with this percentage
or
interest_group_ig_armed_forces_pol_str_mult to increase political strength modifier